src/game/settings.c

```diff
@@ -64,6 +65,8 @@ static void load_default_settings(void)
     data.victory_video = 0;
     data.last_advisor = ADVISOR_LABOR;
 
+    data.digit_separator_pixels = 3;
```

Comment:
Why not directly use the value from `config.c`? It seems you're just copying a value that is already set elsewhere.